homeassistant/components/climate/tado_v1.py

```diff
@@ -0,0 +1,301 @@
+﻿"""tado component to create a climate device for each zone."""
+
+import logging
+
+from homeassistant.const import TEMP_CELSIUS
+
+from homeassistant.components.climate import (
+    ClimateDevice)
+from homeassistant.const import (
+    ATTR_TEMPERATURE)
+from homeassistant.components.tado_v1 import (
+    CONST_TADO_DATA)
+
+CONST_MODE_SMART_SCHEDULE = "SMART_SCHEDULE"  # Default mytado mode
+CONST_MODE_OFF = "OFF"  # Switch off heating in a zone
+
+# When we change the temperature setting, we need an overlay mode
+# wait until tado changes the mode automatic
+CONST_OVERLAY_TADO_MODE = "TADO_MODE"
+# the user has change the temperature or mode manually
+CONST_OVERLAY_MANUAL = "MANUAL"
+# the temperature will be reset after a timespan
+CONST_OVERLAY_TIMER = "TIMER"
+
+
+# will be used when changing temperature
+CONST_DEFAULT_OPERATION_MODE = CONST_OVERLAY_TADO_MODE
```

Comment:
This is not necessary! It is more a private coding style to be less obtrusive, when it comes to later changes. In this case it happened, because I had some other functionality implemented before, but removed it after a while ;=)